app/controllers/admin/facilities_controller.rb

```diff
@@ -70,6 +70,7 @@ def set_facility_group
 
   def facility_params
     params.require(:facility).permit(
+      :facility_group_id,
```

Comment:
Users can modify their form to send any facility group ID. Are we checking that the facility group ID received here is allowed for the current user?

Reply:
Good catch. I'll fix this. We limit the group ID on the form, but you're right that it can be modified.